volume/drivers/coprhd/coprhd.go

```diff
@@ -186,71 +274,205 @@ func (d *driver) Alerts(volumeID string) (*api.Alerts, error) {
 }
 
 func (d *driver) Attach(volumeID string) (path string, err error) {
-	return "", nil
+	volume, err := d.GetVol(volumeID)
+	if err != nil {
+		return "", fmt.Errorf("Volume %s could not be located", volumeID)
+	}
+
+	coprVolume, err := d.client.Volume().
+		WWN(volumeID).
+		Query()
+	if err != nil {
+		return "", err
+	}
+
+	group, err := d.client.Export().
+		Name(coprVolume.Name).
+		Initiators(d.initiator.Id).
+		Volumes(coprVolume.Id).
+		Project(coprVolume.Project.Id).
+		Array(coprVolume.VArray.Id).
+		Create()
+	if err != nil {
+		return "", err
+	}
+
+	protocol := d.initiator.Protocol
+	device := ""
+
+	switch protocol {
```

Comment:
because the assumption is that other initiator protocols supported by coprhd will be implmented in the future.